lib/Conversion/TritonGPUToLLVM/ConvertLayoutOpToLLVM/SharedToDotOperandMFMA.cpp

```diff
@@ -323,22 +323,8 @@ std::optional<int> findConstValue(Value val) {
   return intAttr.getInt();
 }
 
-bool fastPathAvailable(const SharedMemoryObject &smemObj,
-                       const SharedEncodingAttr &srcEncoding,
-                       const MfmaEncodingAttr &dstEncoding) {
-  if (srcEncoding.getMaxPhase() > 1)
-    return false;
-  auto stride0 = findConstValue(smemObj.strides[0]);
-  auto stride1 = findConstValue(smemObj.strides[1]);
-  auto offset0 = findConstValue(smemObj.offsets[0]);
-  auto offset1 = findConstValue(smemObj.offsets[1]);
```

Comment:
To those who are interested in the issue
These are no longer constants due to `stream-pipeline` pass. Therefore, they cannot be used to check is we are going to the fast path.